packages/react/src/scroll-area/root/ScrollAreaRoot.tsx

```diff
@@ -47,15 +37,21 @@ export const ScrollAreaRoot = React.forwardRef(function ScrollAreaRoot(
     ...elementProps
   } = componentProps;
 
+  const overflowEdgeThreshold = normalizeOverflowEdgeThreshold(overflowEdgeThresholdProp);
+
+  const rootId = useBaseUiId();
+
+  const scrollYTimeout = useTimeout();
+  const scrollXTimeout = useTimeout();
+
   const [hovering, setHovering] = React.useState(false);
   const [scrollingX, setScrollingX] = React.useState(false);
   const [scrollingY, setScrollingY] = React.useState(false);
-  const [cornerSize, setCornerSize] = React.useState<Size>(DEFAULT_SIZE);
-  const [thumbSize, setThumbSize] = React.useState<Size>(DEFAULT_SIZE);
+  const [cornerSize, setCornerSize] = React.useState<Dimensions>(DEFAULT_SIZE);
```

Comment:
Nitpick but if we rename `Size` into `Dimensions`, I would also rename the constants and variables to be consistent 👍

Reply:
Probably better just to rename the import to `Size`